packages/ckeditor5-engine/src/model/markercollection.js

```diff
@@ -418,6 +429,19 @@ class Marker {
 		return this._affectsData;
 	}
 
+	/**
+	 * Gets the current marker's data.
+	 *
+	 * @returns {module:engine/model/markercollection~MarkerData}
+	 */
+	getData() {
+		return {
+			range: this.getRange(),
+			affectsData: this.affectsData,
+			managedUsingOperations: this.managedUsingOperations
```

Comment:
Not 100% sure about this property being here, but I can remove it if it does not make sense.